cmd/kubeadm/app/phases/certs/certlist.go

```diff
@@ -21,7 +21,6 @@ import (
 	"crypto/x509"
 
 	"github.com/pkg/errors"
-
```

Comment:
Why delete the blank line? We prefer to use a blank line to distinguish different import blocks.

Reply:
Sorry for bringing our programming style, I would like to ask what tool is used here to automatically control imports？